irohad/validation/chain_validator.hpp

```diff
@@ -55,7 +64,7 @@ namespace iroha {
        * @param storage -  storage that may be modified during block appliance
        * @return true if block is valid and can be applied, false otherwise
        */
-      virtual bool validateBlock(const model::Block &block,
+      virtual bool validateBlock(const shared_model::interface::Block &block,
                                  ametsuchi::MutableStorage &storage) = 0;
     };
   }  // namespace validation
```

Comment:
Why doesn't `validateChain` method change?

Reply:
Dependecy on BlockLoader via Synchronizer (https://soramitsu.atlassian.net/browse/IR-903 and https://soramitsu.atlassian.net/browse/IR-902), it seems better to replace function parameter within these tasks.